pkgs/servers/web-apps/morty/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "morty-${version}";
+  version = "0.1.0";
+
+  goPackagePath = "github.com/asciimoo/morty";
+
+  src = fetchgit {
+    rev = "a680c56daedf34d957032521cbb4b4881b89e13b";
+    url = "https://github.com/asciimoo/morty";
+    sha256 = "0h0yzxmn7z1ygfxf78sfxrinn38w1z6zxz1s06a7p0whzpyxy9ai";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    description = "Privacy aware web content sanitizer proxy as a service";
+    longDescription = "Morty is a web content sanitizer proxy as a
+      service. It rewrites web pages to exclude malicious HTML tags and
+      attributes. It also replaces external resource references to
+      prevent third party information leaks.
+
+      The main goal of morty is to provide a result proxy for searx, but it can be used as a standalone sanitizer service too.
+
+      Features:
+
+      * HTML sanitization
+      * Rewrites HTML/CSS external references to locals
+      * JavaScript blocking
+      * No Cookies forwarded
+      * No Referrers
+      * No Caching/Etag
+      * Supports GET/POST forms and IFrames
+      * Optional HMAC URL verifier key to prevent service abuse
+   ";
```

Comment:
Please use `''` instead of `"` (use two single quotes instead of one double-quote) on long text blocks. Also, move the first line of the text to its own line. Finally, the `maintainer` line should be `maintainers`.